iOS-sample-app/Base.lproj/Main.storyboard

```diff
@@ -38,6 +39,26 @@
                                             <segue destination="XTx-jd-5cc" kind="show" id="cK5-fB-0tf"/>
                                         </connections>
                                     </tableViewCell>
+                                    <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="default" indentationWidth="10" textLabel="AMX-sU-vln" style="IBUITableViewCellStyleDefault" id="OD1-7u-U3s">
```

Comment:
Just a little something. Since Transactions is below ScreenViews in the showcases array, could you make the storyboard a little nicer by having the transactions screen appear below the screenviews reference? Here's what I mean:

<img width="439" alt="Screen Shot 2019-12-31 at 11 24 54" src="https://user-images.githubusercontent.com/9444441/71618572-77180700-2bc0-11ea-8115-96ac4a272416.png">

Reply:
Think of it for all PRs too